src/HonzaBotner/Startup.cs

```diff
@@ -37,6 +38,9 @@ public void ConfigureServices(IServiceCollection services)
             services.AddDatabaseDeveloperPageExceptionFilter();
             services.AddControllers();
 
+            // TODO: Temporary solution - should be rewritten once someone who knows coding stuff sees this!!
+            ulong? guildId = Configuration.GetSection("Discord").GetValue<ulong>("GuildId");
```

Comment:
I need help with this, whoever finds some nicer way to get config guild id into code so early in the startup process, earns zlatého bludišťáka from me.
It is needed at the line 91 when registering slash commads. It might not be needed in production, but due to how Discord handles SlashCommand, it is crucial to register them with GuildId at least for testing. If registered globaly (without guildid), they can take up to 1 hour to sync with discord and show up in clients.

Reply:
ping @ostorc